esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -1282,7 +1282,11 @@ void EsdlBindingImpl::addService(const char * name,
 
 void EsdlBindingImpl::initEsdlServiceInfo(IEsdlDefService &srvdef)
 {
-    const char *verstr = srvdef.queryProp("default_client_version");
```

Comment:
@mayx 

When we discussed offline I didn't realize we did already get the default_client_version from the ESDL... I thought you meant we were only using something from the ESP binding config.

These two tags changed over time and I'm not sure how they are currently used in scapps.  Did you discuss this with Jiafu?  

Schema driven SOAP is more sensitive to interface changes, so originally... 
a SOAP request would have defaulted to the oldest acceptable version... that used to be 0 but default_client_version may override that.  
HTTP GET and FORM post used to default to newest.  Meaning service version.

But again the main thing is that we consider how SCAPPS currently works.